ingest/bin/infer-dengue-serotype.py

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env python3
+
+import argparse
+import json
+from sys import stdin, stdout
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Dengue specific processing of metadata, infer serotype from virus_tax_id"
+    )
+    parser.add_argument(
+        "--virus_tax_id",
```

Comment:
CLI flags are usually hyphen separated, that is the convention we usually follow in Nextstrain. 

I would update to 
```suggestion
        "--virus-tax-id",
```